workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -235,8 +243,18 @@ HttpHeaders.AUTHORIZATION, _getCredentials()
 		return null;
 	}
 
-	private boolean _isPartner() {
-		return true;
+	private boolean _isPartner(Jwt jwt) {
+		ArrayList<String> userRoles = _userAccountService.getUserRoles(jwt);
+
+		if (userRoles.contains(RoleConstants.NAME_ADMINISTRATOR) ||
+			userRoles.contains(RoleConstants.NAME_LIFERAY_STAFF) ||
+			userRoles.contains(RoleConstants.NAME_PARTNER) ||
+			userRoles.contains(RoleConstants.NAME_PROVISIONING_ADMIN)) {
```

Comment:
is there a reason we need Provisioning Admin role checked separately from Liferay Staff?

Reply:
@jennyhchen Felipe mentioned that you were the one to suggest adding this role check for Provisioning Admin. Do we really need this role? Realistically, there should never be a Provisioning Admin that is not also Liferay staff, right?

Reply:
@abfranca can we remove this role for now and just use Liferay Staff then. Seems like Provisioning Admin is not necessary for this use case. Thanks!

Reply:
Done!